vpr/src/place/place.cpp

```diff
@@ -578,7 +696,12 @@ void try_place(const Netlist<>& net_list,
     /* Gets initial cost and loads bounding boxes. */
 
     if (placer_opts.place_algorithm.is_timing_driven()) {
-        costs.bb_cost = comp_bb_cost(NORMAL);
+        if (num_layers == 1) {
+            costs.bb_cost = comp_bb_cost(NORMAL);
+        } else {
+            VTR_ASSERT_SAFE(num_layers > 1);
```

Comment:
If you use cube_bb, aren't the layer = 1 and layer > 1 code paths basically the same?  If so, maybe they can be merged and this test should be on cube_bb instead?